p2p/transport/webrtc/transport.go

```diff
@@ -337,6 +337,7 @@ func (t *WebRTCTransport) dial(
 		t.peerConnectionTimeouts.Failed,
 		t.peerConnectionTimeouts.Keepalive,
 	)
+	settingEngine.SetIncludeLoopbackCandidate(true)
```

Comment:
Is that the reason why it didn't run on my machine?

Does this setting make Pion send loopback addresses over the wire? That would be suboptimal.

Reply:
Not sure. What OS are you on? For me it was working even before this change.
CC @ckousik for the answer on your questions, as it was on his request.

Reply:
I'm on OSX.

Reply:
Next thing I'd ask for is a comment in the code for why we have this setting ;)

Reply:
@ckousik can you help out with this one?

Reply:
FYI it was already set in listener as well before

Reply:
By default, webrtc will not collect candidates on the loopback address. This is disallowed in the ICE specification. However, implementations do not strictly follow this, for eg. Chrome gathers TCP loopback candidates. If you run pion on a system with only the loopback interface UP, it will not connect to anything. However, if it has any other interface (even a private one, eg. `192.168.0.0/16`), it will gather candidates on it and will be able to connect to other pion instances on the same interface.

Reply:
Comment added.